src/EditorFeatures/CSharpTest/Classification/SyntacticClassifierTests.cs

```diff
@@ -3977,5 +3977,38 @@ void M()
                 Number("2"),
                 Punctuation.CloseParen);
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Classification)]
+        public async Task TestConflictMarkers1()
```

Comment:
Sure.  I can add that.  It's more a test that nothing goes horribly wrong (like crashing/index-out-of-bounds) vs actually verifying that we classify correctly.